### PR TITLE
Fix ssl handshake error on erlang 21

### PR DESCRIPTION
**Pull request by arcusfelis**
This PR addresses errors below.

Proposed changes include:
* Remove autogenerated eldap_filter_yecc.erl in make clean 
* Set server_name_indication=disable in mysql_redis preset 

Error 1:

```erlang
===> Compiling src/eldap_filter_yecc.erl failed

../Library/Application Support/ErlangInstaller/21.1/lib/parsetools-2.1.8/include/yeccpre.hrl:59: illegal pattern

../Library/Application Support/ErlangInstaller/21.1/lib/parsetools-2.1.8/include/yeccpre.hrl:60: variable 'Error' is unbound

../Library/Application Support/ErlangInstaller/21.1/lib/parsetools-2.1.8/include/yeccpre.hrl:60: variable 'Stacktrace' is unbound

../Library/Application Support/ErlangInstaller/21.1/lib/parsetools-2.1.8/include/yeccpre.hrl:63: variable 'Stacktrace' unsafe in 'try' (line 60)

../Library/Application Support/ErlangInstaller/21.1/lib/parsetools-2.1.8/include/yeccpre.hrl:64: variable 'Error' unsafe in 'try' (line 60)

../Library/Application Support/ErlangInstaller/21.1/lib/parsetools-2.1.8/include/yeccpre.hrl:64: variable 'Stacktrace' unsafe in 'try' (line 60)
```

Error 2:

```erlang
2019-01-31 13:53:02 =SUPERVISOR REPORT====
     Supervisor: {local,'mongoose_wpool$rdbms$global$default'}
     Context:    start_error
     Reason:     {shutdown,{failed_to_start_child,'wpool_pool-mongoose_wpool$rdbms$global$default-1',{error,{failed_to_upgrade_socket,{tls_alert,"handshake failure"}}}}}
     Offender:   [{pid,undefined},{id,'wpool_pool-mongoose_wpool$rdbms$global$default-process-sup'},{mfargs,{wpool_process_sup,start_link,['mongoose_wpool$rdbms$global$default','wpool_pool-mongoose_wpool$rdbms$global$default-process-sup',[{queue_manager,'wpool_pool-mongoose_wpool$rdbms$global$default-queue-manager'},{time_checker,'wpool_pool-mongoose_wpool$rdbms$global$default-time-checker'},{worker,{mongoose_rdbms,[{server,{mysql,"localhost","ejabberd","ejabberd","mongooseim_secret",[{verify,verify_peer},{cacertfile,"priv/ssl/cacert.pem"}]}}]}},{pool_sup_shutdown,infinity},{workers,5},{overrun_warning,infinity},{max_overrun_warnings,infinity},{overrun_handler,{error_logger,warning_report}},{workers,100},{worker_opt,[]},{queue_type,fifo}]]}},{restart_type,permanent},{shutdown,infinity},{child_type,supervisor}]
```

### CODE REVIEW

File: big_tests/test.config [hunk 1]
```diff
@@ -221,7 +221,9 @@
        {redis, global, default, [{workers, 10}, {strategy, random_worker}], []},
        {rdbms, global, default, [{workers, 5}],
         [{server, {mysql, \"localhost\", \"ejabberd\", \"ejabberd\", \"mongooseim_secret\",
-                   [{verify, verify_peer}, {cacertfile, \"priv/ssl/cacert.pem\"}]}}]}
+                   [{verify, verify_peer}, {cacertfile, \"priv/ssl/cacert.pem\"},
+                    %% To avoid ssl-handshake error in erlang 20 and older
```
Comment (michalwski) [inline big_tests/test.config]: And newer I think. This change affects Erlang 20, 21 and probably future releases.

**Comment by mongoose-im** [comment]
[6159.1](https://travis-ci.org/esl/MongooseIM/jobs/487005105) / Erlang 19.3 / small_tests / 4194bb7749920b916fe53c23315ec9016107f402
Reports [root](http://esl.github.io/mongooseim-ct-reports/s3_reports.html?prefix=PR/2208/6159/small_tests.19.3) / [small](https://mongooseim-ct-results.s3-eu-west-1.amazonaws.com/PR/2208/6159/small_tests.19.3/small/ct_run.mongooseim@localhost.2019-01-31_16.00.58/lib.mongooseim.logs/run.2019-01-31_16.00.59/suite.log.html)

**Comment by mongoose-im** [comment]
[6161.1](https://travis-ci.org/esl/MongooseIM/jobs/487044944) / Erlang 19.3 / small_tests / 922366f610091a5f2f0f4e913f92f0b48c76c86c
Reports [root](http://esl.github.io/mongooseim-ct-reports/s3_reports.html?prefix=PR/2208/6161/small_tests.19.3) / [small](https://mongooseim-ct-results.s3-eu-west-1.amazonaws.com/PR/2208/6161/small_tests.19.3/small/ct_run.mongooseim@localhost.2019-01-31_17.42.53/lib.mongooseim.logs/run.2019-01-31_17.42.55/suite.log.html)

**Comment by arcusfelis** [comment]
too old

**Pull request closed without merging**